### PR TITLE
[14.0][IMP] account_credit_control: Add details (table) to communication mail template

### PR DESCRIPTION
**Pull request by victoralmau**
Add details (table) to communication mail template.
ejemplo

Please @pedrobaeza and @ernestotejeda can you review it?

@Tecnativa TT41337

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please solve conflicts. And the idea is to not have the table embedded in the mail template, but having a check on the credit control run, inject this table in the final content of the mail to be sent.

**Comment by victoralmau** [comment]
Changes done according to the new approach, is this correct?

File: account_credit_control/models/credit_control_communication.py [hunk 1]
```diff
@@ -173,9 +175,43 @@ def _generate_comm_from_credit_lines(self, lines):
         comms._onchange_partner_id()
         return comms
 
+    def _get_credit_control_communication_table(self):
+        th_style = "padding: 5px; border: 1px solid black;"
```
Comment (pedrobaeza) [inline account_credit_control/models/credit_control_communication.py:179]: Put an empty line and a title like "Invoices summary".

File: account_credit_control/models/credit_control_policy.py [hunk 2]
```diff
@@ -249,6 +249,7 @@ class CreditControlPolicyLevel(models.Model):
     email_template_id = fields.Many2one(comodel_name="mail.template", required=True)
     channel = fields.Selection(selection=CHANNEL_LIST, required=True)
     custom_text = fields.Text(string="Custom Message", required=True, translate=True)
+    show_mail_details = fields.Boolean(string="Show Mail details")
```
Comment (pedrobaeza) [inline account_credit_control/models/credit_control_policy.py]:
```suggestion
    mail_show_invoice_detail = fields.Boolean(string="Show Invoice Details in mail")
```

File: account_credit_control/views/credit_control_policy.xml [hunk 3]
```diff
@@ -52,6 +52,7 @@
                                                 colspan="4"
                                             />
                                             <newline />
+                                            <field name="show_mail_details" />
```
Comment (pedrobaeza) [inline account_credit_control/views/credit_control_policy.xml]: Only visible if the action is mail.

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**